src/app/components/employee-profile/employee-profile.component.html

```diff
@@ -0,0 +1,123 @@
+                    </div>
+                </mat-card-content>
+            </mat-card>
+
+            <mat-card class="card2">
+                <mat-card-title>Contact Info
+                    <button mat-icon-button matTooltip="Click to Edit" (click)="edit(0)" class="iconbutton"
+                        color="primary">
+                        <mat-icon aria-label="Edit" class="iconbutton">edit</mat-icon>
+                    </button>
+                </mat-card-title>
+                <mat-divider></mat-divider>
+                <mat-card-content>
+                    <div class="card-1">
+                        <table class="table2">
+                            <tbody>
+                                <thead>
+                                    <tr>
+                                        <th>Work phone:-</th>
+                                        <td>-</td>
+
+                                    </tr>
+                                </thead>
+                                <thead>
+                                    <tr>
+                                        <th>Work email:-</th>
+                                        <td>email</td>
+                                    </tr>
+                                </thead>
+                                <thead>
+                                    <tr>
+                                        <th>Personal phone :-</th>
+                                        <td>9527708661</td>
+                                    </tr>
+                                </thead>
+                                <thead>
+                                    <tr>
+                                        <th>Personal email:-</th>
+                                        <td>email</td>
+                                    </tr>
+                                </thead>
+                            </tbody>
+                        </table>
+                    </div>
+                </mat-card-content>
+            </mat-card>
+        </div>
+
+
+        <!-- <mat-card class="card">
+            <mat-card-title>Addresses
+                <button mat-icon-button matTooltip="Click to Edit" (click)="edit(0)" class="iconbutton" color="primary">
+                    <mat-icon aria-label="Edit" class="iconbutton">edit</mat-icon>
+                </button>
+            </mat-card-title>
+            <mat-divider></mat-divider>
+            <mat-card-content>
+                <div class="card-1">
+                    <table class="table">
+                        <tbody>
+                            <thead>
+                                <tr>
+                                    <th>Home Address:-</th>
+                                    <td>At post:- paldhi tal:- jamner Dist:-</td>
+                                </tr>
+                            </thead>
+
+                        </tbody>
+                    </table>
+                </div>
+            </mat-card-content>
```

Comment:
if need to use in future then add comment for above code else remove commented code